### PR TITLE
Add sidecar resources

### PR DESCRIPTION
**Pull request by blez**

### CODE REVIEW

File: deploy/cr.yaml [hunk 1]
```diff
@@ -96,6 +96,13 @@ spec:
 #    priorityClassName: high-priority
 #    nodeSelector:
 #      disktype: ssd
+    # sidecarResources:
```
Comment (delgod) [inline deploy/cr.yaml]: I know that default commenting is controversion, but can you follow it?
```suggestion
#    sidecarResources:
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| recreate | passed |
| self-healing | failed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| storage | passed |
| upgrade | passed |
| demand-backup | passed |
| limits | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | failed |
| scaling | failed |
| big-data | failed |
| recreate | failed |
| init-deploy | failed |

**Comment by benlangfeld** [comment]
@delgod Might this be included in a release soon?

**Comment by benlangfeld** [comment]
Also, for anyone following along, this appears to be an implementation of https://jira.percona.com/browse/CLOUD-462

**Comment by delgod** [comment]
Hi @benlangfeld 
it is part of CLOUD-462
it will be released in a few weeks.
thanks for the report!

**Pull request merged**